### DISCUSSION TITLE
Replicating --script behavior in v4

### DISCUSSION THREAD

**Discussion by esalter** [mikefarah/yq#833] [Q&A] [answered]
Hello!  I guess this may be a silly question. I read through the "upgrading from v3" document (https://mikefarah.gitbook.io/yq/upgrading-from-v3), but couldn't find an example on how to map using the old v3 flag `--script` to v4. It seems like `eval-all` may do it, and I can see how it works for individual commands, but it's not clear on how exactly how to make it work from an input script file which was very handy.  Do you have somewhere I can look on how to replicate this behavior?  Thank you!

**Comment by mikefarah** [comment, marked as answer]
Atm you can't do that - however in v4 you can have a multiple operations in a single expression, so I guess what you could do is create a bash script file with a single, multi-line expression

```sh
#!/bin/bash

./yq e '
  .a.key1 = "things" |
  .a.ab.key2 = "whatever"
' ./examples/data1.yaml
```

**Reply by mikefarah** [reply]
Might actually add that to the docs
